easybuild/easyconfigs/t/Tensorflow/Tensorflow-1.2.1-foss-2016b-Python-3.5.2.eb

```diff
@@ -0,0 +1,61 @@
+easyblock = 'Bundle'
+
+name = 'Tensorflow'
+version = '1.2.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.tensorflow.org/'
+description = "An open-source software library for Machine Intelligence"
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+pyver = '3.5.2'
+pyshortver = '35'
+
+dependencies = [
+    ('Python', pyver),
```

Comment:
`pyver` is only used in one place, so you can replace this with

```python
('Python', '3.5.2'),
```

and drop the `pyver = ...` line

you'll need to keep the `pyshortver` though, which is why I took a different approach in existing Tensorflow easyconfigs, since now you're risking to forget syncing the `35` with the `3.5.2`

That risk is avoided entirely with an approach like this:

```python
pymaj = '3'
pymin = '6'
pyshortver = pymaj + pymin

dependencies = [
    ('Python', '%s.%s.1' % (pymaj, pymin)),
```

See https://github.com/easybuilders/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/t/Tensorflow/Tensorflow-1.2.0-intel-2017a-Python-3.6.1.eb

Reply:
@boegel 
Sorry!  Was away most of August, and am currently playing catch-up with a deadline Friday.  I will return to this and my other pull requests next week.

Reply:
@boegel 
In this case I am pretty sure I would forget to update the third digit in the Python version.  In fact, I already did when I copied from the 3.6.1 easybuild. :-)

What about this solution:
```
pyver = '3.5.2'
pyshortver = ''.join(pyver.split('.')[:2])
```

Feel free to veto it if it is too obscure!

Reply:
@schiotz makes total sense, let's go with this!